webserver.py

```diff
@@ -33,6 +33,27 @@
 
 class MyHandler(BaseHTTPRequestHandler):
 
+    @staticmethod
+    def _handle_all_instruments(data):
+        active = dict()
+        for key in data:
+            if data[key] != "":
+                active[key] = True
+            else:
+                active[key] = False
+        return str(json.dumps(active))
+
+    @staticmethod
+    def _handle_specific_instrument(instrument, data):
+        if instrument not in data.keys():
+            raise ValueError(str(instrument) + " not known")
+        if data[instrument] == "":
+            raise ValueError("Instrument has become unavailable")
+        try:
+            return str(json.dumps(_scraped_data[instrument]))
+        except Exception as err:
+            raise ValueError("Unable to convert instrument data to JSON: %s" % err.message)
```

Comment:
I'd still like some docs for these 2 methods. That they refer to all instruments and specific instruments is obvious from the contexts in which they're called. The method name therefore only  tells us that these methods "handle" something, which isn't useful.

- Please could they have a method name which describes what the method does
- Please add commenting describing in extra detail what the method does

These two methods are the critical component to how JSON_Bourne communicates with the status screens so I think readability is crucial here. The former returns data for all instruments describing whether IBEX is running on each of them. The latter sends detailed settings on the state of a named instrument. Please could that be obvious from a single glance rather than having to digest the code.